<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>primary</class>
 <widget class="QWidget" name="primary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#primary{background-image:url(:/Log_back.jpg);}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>-5</y>
     <width>901</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stackedWidget{background-image: url(:/back.jpg);}</string>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>150</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget{background-color:  rgb(184, 199, 200);}
</string>
   </property>
   <widget class="QPushButton" name="office_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>科室</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nurse_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>护士</string>
    </property>
   </widget>
   <widget class="QPushButton" name="patience_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;幼圆&quot;;</string>
    </property>
    <property name="text">
     <string>病人</string>
    </property>
   </widget>
   <widget class="QPushButton" name="medicine_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;幼圆&quot;;</string>
    </property>
    <property name="text">
     <string>药品</string>
    </property>
   </widget>
   <widget class="QToolButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;幼圆&quot;;</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doctor_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 15pt &quot;幼圆&quot;;
</string>
    </property>
    <property name="text">
     <string>医生</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
